### ISSUE TITLE
api sessions don't get logged out

### ISSUE THREAD

**Issue by mvernimmen** [dj-wasabi/ansible-zabbix-agent#278]
**Describe the bug**
When running ansible-zabbix-agent, the sessions used to update the zabbix-server are not logged out. This leaves the session state in the sessions table at active ('0'), instead of passive ('1'). When sessions remain active, they are not cleaned up by the zabbix housekeeper. This causes the zabbix sessions table to become very large.

**Installation method/version**
ansible-zabbix-agent role tag 1.5.0 via ansible galaxy
agent is installed as rpm by dj-wasabi role

**Ansible Version**

```
versions between 2.5 and 2.8, most are at 2.7
```

**Targeted hosts**
Concerns the following OS(es):
* CentOS

**Expected behavior**
Sessions are logged out after contacting the zabbix API. Session table entries in zabbix gets set to passive ('1') and get cleaned up by zabbix server.

**Additional context**
in 2 weeks, we got this:
```
mysql> select count(*) as count, userid from sessions group by userid;
+--------+--------+
| count  | userid |
+--------+--------+
|      4 |      1 |
|     25 |      2 |
|      1 |      5 |
|      2 |      9 |
|      1 |     14 |
|      1 |     15 |
|      2 |     23 |
|      1 |     31 |
| 686776 |     36 |
|      2 |     46 |
|      1 |     49 |
|      1 |     50 |
|      2 |     51 |
|      2 |     52 |
|      1 |     53 |
+--------+--------+
```
This is after deleting 11 million rows from that table from previous dj-wasabi runs.

**Comment by dj-wasabi** [comment]
Hi, I created an issue at @ansible : https://github.com/ansible/ansible/issues/63774

**Comment by angystardust** [comment]
@mvernimmen the issue has been fixed in ansible 2.9 (yet to be released) -> https://github.com/ansible/ansible/pull/58525
While we can try to backport the fix to the 2.8/2.7 releases, you should be able to use the module from [devel branch](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/monitoring/zabbix/zabbix_host.py) by the putting it in the `library` folder of your ansible management host

**Comment by mvernimmen** [comment]
Thank you for the quick response and backport, it's greatly appreciated! We'll wait for the fix to be in 2.8 or 2.9 and will clean up the mysql zabbix table periodically until it is there.
@dj-wasabi Besides the changes in ansible, is a change needed to your module also to make use of the logout?

**Comment by dj-wasabi** [comment]
Hi @mvernimmen No changes needed in this role once the PR's are merged in Ansible. Library contains the importing of a module and doing the logout when the library is finished, so no new argument/options needed to use the logout.

**Comment by D3DeFi** [comment]
Hey guys, just to pop in:
- ansible/ansible#63783 - backport was merged. 2.8.7 will include this fixes when released
- ansible/ansible#64735 - original fix, unfortunately, fixed all zabbix modules except zabbix_host. This backport PR should fix that. I really hope this gets merged before 2.9.1 goes out

**Comment by dj-wasabi** [comment]
Thank you @D3DeFi !! 👍
